Makefile

```diff
@@ -41,10 +40,11 @@ DOC_FILES := \
 FIGURE_FILES := \
 	img/media-types.png
 
-EPOCH_TEST_COMMIT ?= v0.2.0
-
 TOOLS := esc gitvalidation glide glide-vc
 
+GO15VENDOREXPERIMENT=1
+export GO15VENDOREXPERIMENT
```

Comment:
My guess is we don't need this anymore right? Go 1.5 was quite a while ago. :sweat_smile:

Reply:
Oh right. 😅

Reply:
lgtm after this is updated :)